File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
# 입력: 문자열
# 52개 리스트 배열 생성
# 알파벳 대문자를 0-25 배열에, 소문자를 26-51에 담는다.
# my_string에서 해당하는 알파벳이 있으면 +1
# 결과값: list로 반환
def solution(my_string):
    # 52개 배열 생성
    answer = [0] * 52
    for i in my_string:
        if(ord(i) - ord('A')) <= 25:
            answer[ord(i) - ord('A')] += 1
        else:
            answer[ord(i) - ord('a') + 26] += 1
    return answer

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    let result = new Array(52).fill(0);
    
    for(let char of my_string){
        if(char >='A' && char <= 'Z'){
            result[char.charCodeAt(0) - 'A'.charCodeAt(0)]++;
        }
        else if(char >= 'a' && char <= 'z'){
            result[char.charCodeAt(0)- 'a'.charCodeAt(0) + 26]++;
        }
    }
    return result;
}